test/integration/suites/upgrade/versions.txt

```diff
@@ -5,3 +5,5 @@
 1.0.0
 1.0.1
 1.0.2
+1.0.3
+1.0.4
```

Comment:
1.0.4 should not be included, since this is 1.0.4.

Reply:
Woof. Thanks!